db/migrate/20190322202902_change_approval_state_column_type.rb

```diff
@@ -0,0 +1,7 @@
+class ChangeApprovalStateColumnType < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :approval_requests, :state, :state_old
```

Comment:
I think we'll need an up and down method just so we can get a `rake db:migration VERSION=` back to the previous state, I'm not seeing how we could ever reset the `state` column to a string here.